manifests/forward.pp

```diff
@@ -5,6 +5,7 @@
 define unbound::forward (
   $address,
   $zone = $name,
+  $forward_first = 'no'
```

Comment:
I'd like to see a validation around this for accepted values of 'yes' and 'no'.  Also please align `=`.